en/pd-recover.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Use PD Recover to Recover the PD cluster
+summary: Learn how to use PD Recover to recover the PD cluster.
+category: reference
+---
+
+# Use PD Recover to Recover the PD cluster
+
+[PD Recover](https://pingcap.com/docs-cn/stable/reference/tools/pd-recover) is a disaster recovery tool of PD, used to recover the PD cluster which cannot start or provide services normally.
+
+## Download PD Recover
+
+1. Download the official TiDB package:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    wget https://download.pingcap.org/tidb-${version}-linux-amd64.tar.gz
+    ```
+
+    In the command above, `${version}` is the version of the TiDB cluster, such as `v4.0.0-rc`.
+
+2. Unpack the TiDB package for installation:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    tar -xzf tidb-${version}-linux-amd64.tar.gz
+    ```
+
+    `pd-recover` is in the `tidb-${version}-linux-amd64/bin` directory.
+
+## Recover the PD cluster
+
+This section introduces how to recover the PD cluster using PD Recover.
+
+### Get Cluster ID
+
+{{< copyable "shell-regular" >}}
+
+```shell
+kubectl get tc ${cluster_name} -n ${namespace} -o='go-template={{.status.clusterID}}{{"\n"}}'
+```
+
+Example:
+
+```
+kubectl get tc test -n test -o='go-template={{.status.clusterID}}{{"\n"}}'
+6821434242797747735
+```
+
+### Get Alloc ID
+
+When you use `pd-recover` to recover the PD cluster, you need to specify `alloc-id` which is larger than the largest allocated ID (`Alloc ID`) of the original cluster.
+
+1. Access the Prometheus monitoring data of the TiDB cluster by taking steps in [Access the monitoring data](monitor-a-tidb-cluster.md#access-the-monitoring-data).
+
+2. Enter `pd_cluster_id` in the input box and click the `Execute` button to make a query. Get the largest value in query result.
+
+3. Multiply the largest value in the query result by `100`. This multiplied value is the `alloc-id` value specified when using `pd-recover`.
+
+### Recover the PD Pod
+
+1. Delete the Pod of the PD cluster.
+
+    Execute the following command to set the value of `spec.pd.replicas` to `0`:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl edit tc ${cluster_name} -n ${namespace}
+    ```
+
+    Because the PD cluster is in an abnormal state, TiDB Operator cannot synchronize the change above to PD StatefulSet. You need to execute the following command to set the value of PD StatefulSet `spec.replicas` to `0`.
```

Comment:
```suggestion
    Because the PD cluster is in an abnormal state, TiDB Operator cannot synchronize the change above to the PD StatefulSet. You need to execute the following command to set the `spec.replicas` of the PD StatefulSet to `0`.
```